### PR TITLE
docs(prisma 2): fix for latest component versions

### LINKED ISSUE #186: Docs with Prisma 2 aren't clear. Throws error.
I am trying https://nexus.js.org/docs/database-access-with-prisma-v2 but the docs aren't clear & throws error.

For example, before `1.3. Add dependencies`, add a `cd myblog/` step which is basic I know but for beginners, it is helpful.

Then Idk why there is `npm init -y` when we already have `package.json` created by `prisma2 init myblog` which I think should be removed as well.

Again `tsconfig.json` is already created by the `prisma2 init myblog` command.

`1.5. Configure start script in package.json` gives another error if I run `yarn start`

Then in `2. Define models & generators` they say edit `project.prisma` but there is no such file. There exists `schema.prisma` in `prisma/` folder. If that is the file we have to edit, then I guess it's already edited.

Let's assume I edited it & run `3. Migrate database`. In that, I run `prisma2 lift save --name="init"` which gives `Everything up-to-date` but running `prisma2 lift up` gives error.

> Error: Error in lift engine for rpc migrationProgress:
  thread 'main' panicked at 'Could not load migration from database. Migration name was: 20190624162221-init', src/libcore/option.rs:1034:5

It basically is terrible at this point. I don't even know if I'm doing it correctly. It throws a ton of errors. Docs has a lot of problem. Would love if it's been edited properly so it works fine.

Unfortunately, I haven't made it work so can't send a PR.

### PR DESCRIPTION
**Pull request by jasonkuhrt**
Will fix #186 
Will fix prisma/nexus-prisma#380
Will fix prisma/nexus-prisma#360

Drastic simplification. Reduced to a teaser basically. Relies on the example in `nexus-prisma` repo to get people going practically, rather than trying to write it out step by step in prose. Motivation is to reduce present maintenance burden while we focus on actually delivering the promise this quarter. Also includes some succinct production overview, which was there before, but again minimal.

### CODE REVIEW

File: docs/database-access-with-prisma-v2.md [hunk 1]
```diff
@@ -8,565 +8,71 @@ This page explains how to use [Prisma](https://github.com/prisma/prisma) and the
...
 ## Overview
 
-### What is Prisma?
+## What is Prisma?
 
-Prisma is a replacement for traditional ORMs. It enables simplified and type-safe database access through [Photon](https://photonjs.prisma.io/). Photon is an auto-generated and type-safe database client. Combined with GraphQL Nexus, it provides a powerful solution for implementing database access in GraphQL resolvers.
+Prisma is a toolchain replacing traditional ORMs. Developers leverage it to drive their database schemas, migrations, type-safe client access, and more.
 
-Prisma currently supports the following databases:
+Developers use the Prisma Definition Language (PDL) to define their database schema. They use [Photon](https://photonjs.prisma.io/) to derive (generate) type-safe database clients that access that databse. As they invariably evolve their database schema, [Lift](https://lift.prisma.io/) is there to carry out complex migrations as needed.
```
Comment (deadcoder0904) [inline docs/database-access-with-prisma-v2.md]:
```suggestion
Developers use the Prisma Definition Language (PDL) to define their database schema. They use [Photon](https://photonjs.prisma.io/) to derive (generate) type-safe database clients that access that database. As they invariably evolve their database schema, [Lift](https://lift.prisma.io/) is there to carry out complex migrations as needed.
```

File: docs/database-access-with-prisma-v2.md [hunk 2]
```diff
@@ -8,565 +8,71 @@ This page explains how to use [Prisma](https://github.com/prisma/prisma) and the
...
-
-> `UserCreateInput` and `UserUpdateInput` differ in the way relation fields are treated.
+[`nexus-prisma`](https://github.com/prisma/nexus-prisma) is a plugin for Nexus. It extends the Nexus DSL making it easy to expose Prisma models and operations against them in your GraphQL schema. Specifically the Nexus `t` is extended with `.model` and `.crud`. The former assists with exposing your Prisma Models as GraphQL Objects. The latter is valid within `Mutation` and `Query` Nexus type blocks and helps you expose operations over your Prisma Models including rich input types to express pagination, ordering, filtering and so forth.
 
 ## Examples
 
-Here's a minimal example for using `nexus-prisma`:
-
-**Prisma schema**:
+You can find a runnable `nexus-prisma` example in the [`nexus-prisma repo`](https://github.com/prisma/nexus-prisma/tree/next/example). The following are isolated snippets to give you a sense for the DSL.
```
Comment (deadcoder0904) [inline docs/database-access-with-prisma-v2.md]: [`nexus-prisma repo`](https://github.com/prisma/nexus-prisma/tree/next/example) not sure if its purposely done but it's 404

**Comment by Mentioum** [comment]
Right now as far as I can tell there isn't a single working example for prisma2 and prisma nexus.

**Comment by jasonkuhrt** [comment]
Hey @Mentioum the example should work. Feel free to reach out on slack and I would be more than happy to help you debug. Also interested in seeing what features we can add to enable the pit of success.

**Comment by jasonkuhrt** [comment]
Superseded by https://github.com/prisma-labs/nexus-prisma/pull/459

**Pull request closed without merging**